bzt/modules/apiritif/generator.py

```diff
@@ -1077,6 +1087,64 @@ def _get_remote_webdriver(self):
                         arg="options",
                         value=ast.Name(id="options"))]))
 
+    def _get_selenium_options(self):
+        options = "options"
+        if options in self.executor.settings:
+            self.log.debug("Generating selenium options")
+            return [self._get_ignore_proxy() + self._get_arguments()]
+        return []
+
+    def _get_ignore_proxy(self):
+        options = "options"
+        ignore_proxy = "ignore_proxy"
+        if ignore_proxy in self.executor.settings[options]:
+            if self.executor.settings[options][ignore_proxy] is True:
+                return [ast.If(
```

Comment:
add warning for selenium 3 here

Reply:
https://github.com/Blazemeter/taurus/blob/f125ffcb6308b74452ace61dc6006bb0d7769638/bzt/utils.py#L168